src/infra/openTelemetry/tracing.service.ts

```
@@ -0,0 +1,45 @@
+import { resourceFromAttributes } from '@opentelemetry/resources';
+
+@Injectable()
+export class TracingService implements OnModuleInit, OnModuleDestroy {
+    private sdk: NodeSDK;
+
+    private readonly logger = new CustomWinstonLogger();
+
+    async onModuleInit(): Promise<void> {
+        this.sdk = new NodeSDK({
+            resource: resourceFromAttributes({
+                [ATTR_SERVICE_NAME]: process.env.TRACE_SERVICE_NAME || 'nestjs-app',
```

ESLint

[error] Insert `⏎···················`

(prettier/prettier)